modules/partition-replicator/src/main/java/org/apache/ignite/internal/partition/replicator/PartitionReplicaLifecycleManager.java

```diff
@@ -469,13 +468,15 @@ private void cleanUpResourcesForDroppedZonesOnRecovery() {
         // TODO: IGNITE-20384 Clean up abandoned resources for dropped zones from vault and metastore
     }
 
-    private CompletableFuture<Void> onCreateZone(CreateZoneEventParameters createZoneEventParameters) {
+    private CompletableFuture<Boolean> onCreateZone(CreateZoneEventParameters createZoneEventParameters) {
```

Comment:
Why?

Reply:
To have ability to declare the listener simply using a method reference:

`private final EventListener<CreateZoneEventParameters> onCreateZoneListener = this::onCreateZone;`

Because `EventListener`'s method requires `CompletableFuture<Boolean>`